docs/whats_new.rst

```diff
@@ -93,13 +93,22 @@ Research Areas
 FedSM
 -----
 The `FedSM example <https://github.com/NVIDIA/NVFlare/blob/main/research/fed-sm/README.md>`_ illustrates the personalized federated learning algorithm `FedSM <https://arxiv.org/abs/2203.10144>`_
-accepted to CVPR2022. It bridges the different data distributions across clients via a SoftPull mechanism and utilizes
+accepted to CVPR 2022. It bridges the different data distributions across clients via a SoftPull mechanism and utilizes
 a Super Model. A model selector is trained to predict the belongings of a particular sample to any of the clients'
 personalized models or global model. The training of this model also illustrates a challenging federated learning scenario
 with extreme label-imbalance, where each local training is only based on a single label towards the optimization for
 classification of a number of classes equivalent to the number of clients. In this case, the higher-order moments of the
 Adam optimizer are also averaged and synced together with model updates.
 
+Auto-FedRL
+----------
+The `Auto-FedRL example <https://github.com/NVIDIA/NVFlare/blob/main/research/auto-fed-rl/README.md>`_ implements the automated machine learning solution described in
+`Auto-FedRL: Federated Hyperparameter Optimization for Multi-institutional Medical Image Segmentation <https://arxiv.org/abs/2203.06338>`_ accepted to ECCV 2023.
```

Comment:
The conference name should be ECCV 2022.

Reply:
Good catch! Changed.